client/src/Review.jsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import StarIcon from "./StarIcon.jsx";
+import VerifiedIcon from "./VerifiedIcon.jsx";
+
+const Review = props => {
+  //depends on review score, only first few star will be need to be filled
+  let filedStars = [];
+  let numOfstar = props.review.star;
+  let numOfUnfiledStar = 5 - numOfstar;
+  let unfiledStars = [];
+  for (let i = 0; i < numOfstar; i++) {
+    filedStars.push(<StarIcon fill key={i} />);
+  }
+  for (let j = 0; j < numOfUnfiledStar; j++) {
+    unfiledStars.push(<StarIcon key={j + numOfstar} />);
+  }
+  let stars = filedStars.concat(unfiledStars);
```

Comment:
I see why you use concat to merge `filedStars` and `unfiledStars`, but I actually don't think you need both of those arrays. What if you started by creating one single array called `stars`. Then in your first for loop you push your filled stars into it and in your second for loop you push your unfilled stars into it? That would reduce the number of variables that you need to make and it would also save you the trouble of having to concatenate arrays.

Reply:
thank you for your suggestion! It looks much better :)